x-pack/plugins/stack_alerts/public/alert_types/es_query/expression/search_source_expression_form.tsx

```diff
@@ -227,11 +222,7 @@ export const SearchSourceExpressionForm = (props: SearchSourceExpressionFormProp
 
       <EuiSpacer size="s" />
 
-      <DataViewSelectPopover
-        dataViewName={dataView?.getName?.() ?? dataView?.title}
-        dataViewId={dataView?.id}
-        onSelectDataView={onSelectDataView}
-      />
+      <DataViewSelectPopover dataView={dataView} onSelectDataView={onSelectDataView} />
```

Comment:
Once a rule is created via ad hoc data view, it's not possible to see the details of its configuration (which index pattern? which timefield?). Do we have anything in plans to improve this UX?

<img width="1724" alt="Screenshot 2022-10-18 at 13 24 32" src="https://user-images.githubusercontent.com/1415710/196416871-d15fd301-052b-4d5a-b9d7-e93e92696154.png">

Reply:
I think it might be benefitial in the furure to add `Manage this data view` button, but not in the scope of this PR. WDYT @elastic/kibana-data-discovery?

Reply:
definitely! It's not in the scope of this PR. But it's a good catch, that there's actually no information available about which indices are being queried. At least when hovering it a title tag or something similar would be helpful. it's a unified search related topic, right @stratoula ?

Reply:
@kertal it depends :D Which component are you using?

Reply:
We use [`DataViewList`](https://github.com/elastic/kibana/blob/a670c7f37651cbccacb9b054b622ad308d3eec8d/src/plugins/unified_search/public/dataview_picker/dataview_list.tsx#L1) from unifiedSearch.